Week-5/Assignment-5.4/Assignment-5.4.test.js

```diff
@@ -0,0 +1,36 @@
+const mathOperations = require("./Assignment-5.4");
+
+describe("Math Operations Test: ", ()=>{
+    test("Test for addition of 1 + 2 should return 3 ",()=>{
+        expect(mathOperations.sum(1,2)).toBe(3);
```

Comment:
[For your learning] Add some negative test cases as well.. for ex : if you pass string to add method as in sum('1','2') result will be 12 right.. so add a functionality in the code such that data type is only numbers.. if the parameters are integers return "number are not integers".. add this test case as well..